corehq/apps/export/templates/export/customize_export.html

```diff
@@ -548,22 +559,30 @@
                                         {% if request|toggle_enabled:'SPLIT_MULTISELECT_EXPORT' %}
-                                        <td class="input" data-bind="if: allOptions">
-                                            <span data-bind="if: hasNewOptions" class="pull-right">
-                                                <span class="icon-asterisk" title="{% trans "New options available" %}"></span>
-                                            </span>
-                                            <select data-bind="
-                                                value: doc_type,
-                                                foreach: $root.column_type_options
-                                            ">
-                                                <option data-bind="value: value, text: label"></option>
-                                            </select>
-                                            <div data-bind="if: doc_type() === 'SplitColumn'">
-                                                <select class="select2" data-bind="options: allOptions, selectedOptions: options" multiple="true">
+                                        <td class="input">
+                                            <div data-bind="if: !allOptions()">
+                                                <select disabled>
+                                                    <option>{% trans "plain" %}</option>
+                                                </select>
+                                            </div>
+                                            <div data-bind="if: allOptions">
+                                                <span data-bind="if: hasNewOptions" class="pull-right">
+                                                    <span class="icon-asterisk" title="{% trans "New options available" %}"></span>
+                                                </span>
+                                                <select data-bind="
+                                                    value: doc_type,
+                                                    foreach: $root.column_type_options
+                                                "/>
+                                                    <option data-bind="value: value, text: label"></option>
                                                 </select>
-                                                <br/>
-                                                <input class="input input-small" type="text" data-bind="value: newOption"/>
-                                                <button class="btn" data-bind="click: addOption">{% trans "Add" %}</button>
+                                                <span data-bind="makeHqHelp: {name: '{% trans "Split multi-select data" %}', 'placement': 'left'}"></span>
+                                                <div data-bind="if: doc_type() === 'SplitColumn'">
```

Comment:
Same here. `doc_type` is being called, while [above](https://github.com/dimagi/commcare-hq/pull/5647/files#diff-cca39dcdc416953577a5563f252e7d84R573) it's being used as a value. I'm beginning to wonder if Knockout bindings have entirely different semantics than JavaScript (if so, how confusing).